backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceType.java

```diff
@@ -81,35 +84,23 @@ public DeviceType(
     this.supportedDiseaseTestPerformed = supportedDiseaseTestPerformed;
   }
 
-  public String getName() {
-    return name;
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    DeviceType that = (DeviceType) o;
+    return Objects.equals(model, that.model)
+        && Objects.equals(name, that.name)
+        && Objects.equals(testLength, that.testLength)
+        && Objects.equals(manufacturer, that.manufacturer);
   }
 
-  public void setName(String name) {
-    this.name = name;
-  }
-
-  public String getManufacturer() {
-    return manufacturer;
-  }
-
-  public void setManufacturer(String manufacturer) {
-    this.manufacturer = manufacturer;
-  }
-
-  public String getModel() {
-    return model;
-  }
-
-  public void setModel(String model) {
-    this.model = model;
-  }
-
-  public int getTestLength() {
-    return this.testLength;
-  }
-
-  public void setTestLength(int testLength) {
-    this.testLength = testLength;
+  @Override
```

Comment:
Had to add these equal / hashcode overrides to get the tests passing so went ahead and also added / cleaned up the getter / setter methods here